repos/system_upgrade/el7toel8/models/repositoriesfacts.py

```diff
@@ -25,3 +26,19 @@ class RepositoriesFacts(Model):
     topic = SystemFactsTopic
 
     repositories = fields.List(fields.Model(RepositoryFile))
+
+
+@deprecated(
+    since="2020-02-01",
```

Comment:
Do we really want to cheat the deprecation process this way? I'd just use the current date.

Reply:
@bocekm I was trying to follow the draft version of the docs related to the deprecation process (https://github.com/pirat89/leapp/commit/92690c36c2b61b7ce09eb268e839b504d8d1806d#diff-42e97d6a6a1f32cee83f15fb67893b98R17-R18) .  Thus I added this minimum 6 month period. However, there is no difference, wether it is today or today + 180 days. both opttions will result in the report with the severity HIGH (https://github.com/oamg/leapp/commit/b8f11beb060b589c88ac6e93479e357b0607889a#diff-46ef8e2c58eae2fcd84596e29d65b614R302-R303)

Reply:
@ZhukovGreen From my POV, I do not see reason to set any lower risk during the 6m period - as after 6m period the functionality will be whenever dropped. I will just update the documentation probably. But it can be discussed e.g. on cabal mtg. @oamg/developers

Reply:
ha. I have forgotten to put the topic on our cabal mtg. my bad :( I will do it now

Reply:
I'm ok with leaving the functionality in place for 6 months, so I'd use the "current" date, e.g. 2020-09-01. Let's deal with it now to not block the PR. Waiting for cabal would be just wasting last days before the final release.